### PR TITLE
Add option to not sort unprocessed results

### LINKED ISSUE #98: Optional sorting when outputUnprocessed: true
Currently using the outputUnprocessed flag for dev environments, which is awesome. Only issue is some bundles require loading files in a particular order - is it possible to add an option (something like sortUnprocessed), to optionally disable the alpha-order sorting?

### PR DESCRIPTION
**Pull request by juliancarrivick**
Resolves #98 

- Default to sorting to preserve backwards compatibility
- Add unit and integration tests
- Update documentation

### CODE REVIEW

**Comment by chmontgomery** [comment]
Thanks for the PR! I'll look through it soon and get it merged.

File: docs/API.md [hunk 1]
```diff
@@ -198,3 +199,12 @@ Type `String`
...
 Like `options.pathPrefix`, but applies to the unprocessed file result.
+
+### options.sortUnprocessedOutput
+
+Type `Boolean`
+
+Default: `true`
+
+When `true`, will sort unprocessed output in alphanumeric order, otherwise will be in source
+input as specified in the bunld config.
```
Comment (chmontgomery) [inline docs/API.md]: typo

File: lib/results/index.js [hunk 2]
```diff
@@ -8,14 +8,15 @@ var through = require('through2'),
   _ = require('lodash'),
   addBundleResults = require('./add-to-results');
 
-function defaulOptions(opts) {
+function defaultOptions(opts) {
```
Comment (chmontgomery) [inline lib/results/index.js]: thanks for finding and fixing this!

Comment (juliancarrivick) [inline lib/results/index.js]: No problem :)

File: docs/API.md (hunk 1 above)
Comment (juliancarrivick) [inline docs/API.md]: Fixed!

**Comment by nathanwoulfe** [comment]
Deadset legend.

**Pull request merged**